airflow/www/static/js/graph.js

```diff
@@ -598,7 +601,7 @@ function focusGroup(nodeId) {
 }
 
 // Expands a group node
-function expandGroup(nodeId, node, focus = true) {
+function expandGroup(nodeId, node, focus = true, do_draw = true) {
```

Comment:
Could we change the variable to `shouldDraw`?  js should be camelCased and I'm not sure if `do` is a vague verb.

Reply:
Fixed.